receptor_affinity/debugger.py

```diff
@@ -1,39 +1,16 @@
 import asyncio
 import json
-import socket
 import time
 import uuid
 import dateutil.parser
 
 import click
 from aiohttp import web
-
 from receptor import Controller
 from receptor import ReceptorConfig
 from receptor.exceptions import UnrouteableError
 
-
-def random_port(tcp=True):
-    """Get a random port number for making a socket
-
-    Args:
-        tcp: Return a TCP port number if True, UDP if False
-
-    This may not be reliable at all due to an inherent race condition. This works
-    by creating a socket on an ephemeral port, inspecting it to see what port was used,
-    closing it, and returning that port number. In the time between closing the socket
-    and opening a new one, it's possible for the OS to reopen that port for another purpose.
-
-    In practical testing, this race condition did not result in a failure to (re)open the
-    returned port number, making this solution squarely "good enough for now".
-    """
-    # Port 0 will allocate an ephemeral port
-    socktype = socket.SOCK_STREAM if tcp else socket.SOCK_DGRAM
-    s = socket.socket(socket.AF_INET, socktype)
-    s.bind(("", 0))
-    addr, port = s.getsockname()
-    s.close()
-    return port
+from .utils import random_port
```

Comment:
Is a relative import used to avoid the following verbose statement?

```python
from receptor_affinity.utils import random_import
```

It's totally good to do, but I'm just curious as to the reasoning.

Reply:
Before moving the code to this repo we were using it as helper modules on the receptor `test/perf` directory and we used relative imports to be easier to handle them. (this was to provide some more context)

I saw some other places that the relative still being used on this repo and therefore I kinda followed the same pattern.

Reply:
Gotcha. Thanks!